main/src/modules/reactiveLayer/behaviorManager/src/speech.cpp

```diff
@@ -0,0 +1,19 @@
+#include "speech.h"
+
+using namespace std;
+using namespace yarp::os;
+using namespace yarp::sig;
+
+void Speech::configure() {
+    ;
+}
+
+void Speech::run(Bottle args) {
+    yInfo() << "Speech::run";
+    // Bottle *objects = sensation.get(1).asList();
+    // Might want to include gestures like looking at partner and waving a hand
+
+    iCub->say(args.get(0).asString());
```

Comment:
Currently, the `behaviorManager` interprets information and sends a bottle in the following manner to the behavior: `(arg1 arg2)`. I think to be compatible with that, this line should be `iCub->say(args.get(0).asList()->get(0).asString())`.

The corresponding `rpc` command to the behaviorManager would be `speech ("Hello world, iCub speaking")`.